### PR TITLE
Layouts-spotify-Assingment-project

### PR DESCRIPTION
**Pull request by mugayamadox**
This is my initial submission for the spotify project.

### CODE REVIEW

**Review by edwinmp — CHANGES_REQUESTED** [review]
Great job on this.
Now, lets get you out of the ID habit. You next review request should not have a single ID that you can't explain as necessary.
Also give your body some styles so it's not affected by default browser styles e.g. my Chrome & Firefox give your body a default margin of 8px.

**Comment by mugayamadox** [comment]
> Great job on this.
> Now, lets get you out of the ID habit. You next review request should not have a single ID that you can't explain as necessary.
> Also give your body some styles so it's not affected by default browser styles e.g. my Chrome & Firefox give your body a default margin of 8px.

Hi there Mr. Edwin

So on this issue of IDs. 
Honestly as far as my layout  implementation skills are concerned, 
Each ID I used was necessary.
Though I am sure that with further knowledge of layout techniques maybe one can use fewer IDs. And avoid the clutter in my submission.
 
Inorder to adjust, I need your recommendations on how I can improve on my DRY abilities and suggestions on a sample of what would be a better implementation. With that, I guess I fill the gaps.
Thanks

**Comment by edwinmp** [comment]
> Inorder to adjust, I need your recommendations on how I can improve on my DRY abilities and suggestions on a sample of what would be a better implementation. With that, I guess I fill the gaps.

Simple. Instead of an ID, use a CSS class. There's no mysterious knowledge to it. Your ID even have proper names that perfectly fit classes, so just use classes.

So, instead of `#wrapper`, use `.wrapper` and update usages from `<div id='wrapper'></div>` to `<div class="wrapper"></div>`. I'm sure you already knew how to do that, but just wanted to repeat it to show u that what you're doing is a habit and not a necessity.

**Comment by mugayamadox** [comment]
> > Inorder to adjust, I need your recommendations on how I can improve on my DRY abilities and suggestions on a sample of what would be a better implementation. With that, I guess I fill the gaps.
> 
> Simple. Instead of an ID, use a CSS class. There's no mysterious knowledge to it. Your ID even have proper names that perfectly fit classes, so just use classes.
> 
> So, instead of `#wrapper`, use `.wrapper` and update usages from `<div id='wrapper'></div>` to `<div class="wrapper"></div>`. I'm sure you already knew how to do that, but just wanted to repeat it to show u that what you're doing is a habit and not a necessity.

Alright,
That clears all my inquiries.
Let me adjust my code

File: index.html [hunk 1]
```diff
@@ -6,13 +6,85 @@
...
+    <link href="styles/style.css" rel="stylesheet" />
   </head>
   <body>
-    Premium Discover Help Download Music for everyone. Spotify is now free on mobile, tablet and computer. Listen to the
-    right music, wherever you are. What’s on Spotify? Millions of Songs There are millions of songs on Spotify HD Music
-    Listen to music as if you were listening live Stream Everywhere Stream music on your smartphone, tablet or computer
-    It’s as yeezy as Kanye West. Search Know what you want to listen to? Just search and hit play. Browse Check out the
-    latest charts, brand new releases and great playlists for right now. Discover Enjoy new music every Monday with your
-    own personal playlist. Or sit back and enjoy Radio.
+    <div id="nav">
```
Comment (edwinmp) [inline index.html]: Make semantic and replace with a `nav` element instead of a div.

File: index.html [hunk 2]
```diff
@@ -6,13 +6,85 @@
...
+      <img class="logo" src="images/spotify-logo.png" />
+      <ul>
+        <li>Premium</li>
+        <li>Discover</li>
+        <li>Help</li>
+        <li>Download</li>
+      </ul>
+    </div>
+
+    <div id="landing">
```
Comment (edwinmp) [inline index.html]: Demonstrate semantic HTML knowledge and replace these with `<section>` elements.

**Review by edwinmp — CHANGES_REQUESTED** [review]
We'll spare some time to review your implementation in our chat tomorrow so I can demonstrate DRY in action for your CSS.

File: index.html (hunk 2 above)
Comment (mugayamadox) [reply index.html]: Ohkay
I can do that

**Comment by edwinmp** [comment]
@mugayamadox what's the status of the responsiveness?

**Pull request closed without merging**